plugins/woocommerce/tests/php/includes/class-wc-tax-test.php

```
@@ -700,4 +701,315 @@ private function create_tax_rate_for_class( $tax_class = '', $rate = 20.0 ) {
+			'tax_rate_country'  => 'GB',
+			'tax_rate_state'    => '',
+			'tax_rate'          => '20.0000',
+			'tax_rate_name'     => 'VAT',
+			'tax_rate_priority' => '1',
+			'tax_rate_compound' => '0',
+			'tax_rate_shipping' => '1',
+			'tax_rate_order'    => '1',
+			'tax_rate_class'    => '',
+		);
+
+		$tax_rate_id = WC_Tax::_insert_tax_rate( $tax_rate );
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$tax_rate_id'. (undefined)

(UnusedLocalVariable)